docs/conf.py

```diff
@@ -97,7 +97,7 @@
 # The short X.Y version.
 version = "0.9.0"
 # The full version, including alpha/beta/rc tags.
-release = "0.9.0-rc.4"
+release = "0.9.0"
```

Comment:
Don't need the rc anymore?

Reply:
good to release. Kris is happy with this version